<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmOperations</class>
 <widget class="QMainWindow" name="frmOperations">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Ajustes Contables</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QWidget" name="">
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>&amp;Buscar</string>
          </property>
          <property name="buddy">
           <cstring>txtSearch</cstring>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="txtSearch"/>
        </item>
        <item row="1" column="0" colspan="2">
         <widget class="QTableView" name="tableNavigation">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>1</verstretch>
           </sizepolicy>
          </property>
          <property name="editTriggers">
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
          <property name="alternatingRowColors">
           <bool>true</bool>
          </property>
          <property name="selectionMode">
           <enum>QAbstractItemView::SingleSelection</enum>
          </property>
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectRows</enum>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QGroupBox" name="groupBox">
          <property name="sizePolicy">
           <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
            <horstretch>1</horstretch>
            <verstretch>1</verstretch>
           </sizepolicy>
          </property>
          <property name="title">
           <string>Detalles</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_3">
           <item row="0" column="0">
            <widget class="QLabel" name="label">
             <property name="text">
              <string>&amp;Fecha</string>
             </property>
             <property name="buddy">
              <cstring>dtPicker</cstring>
             </property>
            </widget>
           </item>
           <item row="0" column="2">
            <widget class="QDateTimeEdit" name="dtPicker">
             <property name="readOnly">
              <bool>true</bool>
             </property>
             <property name="calendarPopup">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="0" column="3">
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>Concepto</string>
             </property>
             <property name="buddy">
              <cstring>cbConcepts</cstring>
             </property>
            </widget>
           </item>
           <item row="0" column="4">
            <widget class="QStackedWidget" name="stConcepts">
             <property name="currentIndex">
              <number>0</number>
             </property>
             <widget class="QWidget" name="page_3">
              <layout class="QHBoxLayout" name="horizontalLayout">
               <item>
                <widget class="QLineEdit" name="txtConcept">
                 <property name="alignment">
                  <set>Qt::AlignCenter</set>
                 </property>
                 <property name="readOnly">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
             <widget class="QWidget" name="page_4">
              <layout class="QHBoxLayout" name="horizontalLayout_2">
               <item>
                <widget class="QComboBox" name="cbConcepts">
                 <property name="enabled">
                  <bool>false</bool>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QTableView" name="tableDetails">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="MinimumExpanding">
            <horstretch>1</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="editTriggers">
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
          <property name="alternatingRowColors">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNuevo"/>
   <addaction name="actionGuardar"/>
   <addaction name="actionCancelar"/>
  </widget>
  <action name="actionNuevo">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/res/document-new.png</normaloff>:/icons/res/document-new.png</iconset>
   </property>
   <property name="text">
    <string>nuevo</string>
   </property>
   <property name="toolTip">
    <string>Crear nuevo Ajuste Contable</string>
   </property>
  </action>
  <action name="actionGuardar">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/res/document-save.png</normaloff>:/icons/res/document-save.png</iconset>
   </property>
   <property name="text">
    <string>guardar</string>
   </property>
  </action>
  <action name="actionCancelar">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/res/dialog-cancel.png</normaloff>:/icons/res/dialog-cancel.png</iconset>
   </property>
   <property name="text">
    <string>Cancelar</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
